setup.py

```diff
@@ -31,6 +33,16 @@
     "pyyaml",
 ]
 
+
+class PostInstallCommand(install):
```

Comment:
`InstallWithPostCommand` or something would make it clearer that this is actually _also_ running the normal installation